code/modules/customitems/item_defines.dm

```diff
@@ -1348,4 +1348,20 @@
 	righthand_file = 'icons/mob/inhands/fluff_righthand.dmi'
 	icon_state = "teri_horn"
 	item_state = "teri_horn"
-	honk_sound = 'sound/items/teri_horn.ogg'
\ No newline at end of file
+	honk_sound = 'sound/items/teri_horn.ogg'
+
+/obj/item/clothing/suit/fluff/vetcoat //Furasian: Fillmoore Grayson
+    name = "Veteran Coat"
+    desc = "A old, yet well-kept Nanotrasen uniform. Very few of its kind are still produced."
```

Comment:
Should this say "An old", unless this is intentional?

Reply:
"An" would be correct in that case (use an if the following word begins with a vowel is general rule of thumb)